mojave/machinery/machinery.dm

```diff
@@ -110,17 +109,71 @@
 
 // Buttons //
 
-/obj/machinery/button/door/ms13
+/obj/machinery/button/ms13
```

Comment:
As Original explained in Discord, there is a reason these were pathed the way they were before.

Reply:
Last I checked, Buttons completely ignored shutters when not a subtype of the door button. Go test it and double check before you do this. Shutters are essentially the _ONLY_ thing that buttons or terminals are used for, basically.

Reply:
functionality has been added to the base button of ours so we can add it to more than just doors/shutters in the future